monkey/infection_monkey/island_api_client/http_island_api_client.py

```diff
@@ -62,7 +62,7 @@ def connect(
     ):
         try:
             self._http_client.server_url = f"https://{island_server}/api/"
-            self._http_client.get(params={"action": "is-up"})
+            self._http_client.get(params={"action": "is-up"}, disable_retries=True)
```

Comment:
```suggestion
            requests.get("https://{island_server}/api/", params={"action": "is-up"})
```

Reply:
`HTTPClient` is the only place in the Agent which makes requests. I don't think we want to break that boundary.